bitnami/influxdb/templates/NOTES.txt

```diff
@@ -80,62 +80,67 @@ To connect to your database from outside the cluster execute the following comma
 
   {{- if eq .Values.architecture "high-availability" }}
 
   (write operations):
 
-    export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "influxdb.fullname" . }}-relay -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.adminUser.name }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host $SERVICE_IP -port {{ .Values.relay.service.port }}
+    export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "common.names.fullname" . }}-relay -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.auth.admin.username }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host $SERVICE_IP -port {{ .Values.relay.service.port }}
 
   (read operations):
 
   {{- end }}
 
-    export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "influxdb.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.adminUser.name }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host $SERVICE_IP -port {{ .Values.influxdb.service.port }}
+    export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "common.names.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.auth.admin.username }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host $SERVICE_IP -port {{ .Values.influxdb.service.port }}
 
 {{- else if contains "ClusterIP" .Values.influxdb.service.type }}
 
   {{- if eq .Values.architecture "high-availability" }}
 
   (write operations):
 
-    kubectl port-forward svc/{{ include "influxdb.fullname" . }}-relay 9096:{{ .Values.relay.service.port }} &
-    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.adminUser.name }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host 127.0.0.1 -port 9096
+    kubectl port-forward svc/{{ include "common.names.fullname" . }}-relay 9096:{{ .Values.relay.service.port }} &
+    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.auth.admin.username }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host 127.0.0.1 -port 9096
 
   (read operations):
 
   {{- end }}
 
-    kubectl port-forward svc/{{ include "influxdb.fullname" . }} 8086:{{ .Values.influxdb.service.port }} &
-    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.adminUser.name }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host 127.0.0.1 -port 8086
+    kubectl port-forward svc/{{ include "common.names.fullname" . }} 8086:{{ .Values.influxdb.service.port }} &
+    {{- if .Values.authEnabled }}INFLUX_USERNAME="{{ .Values.auth.admin.username }}" INFLUX_PASSWORD="$ADMIN_PASSWORD"{{- end }} influx -host 127.0.0.1 -port 8086
 
 {{- end }}
 
 {{- include "influxdb.validateValues" . }}
-{{- include "influxdb.checkRollingTags" . }}
+
```

Comment:
Should we add a test to ensure **INFLUX_PASSWORD** is set during upgrades?

Reply:
Sure, I completely forgot about it, good catch!!